contrib/docker/docker-compose.yml

```diff
@@ -29,20 +31,22 @@ services:
       POSTGRES_PORT: 5432
       REDIS_HOST: redis
       REDIS_PORT: 6379
-      SUPERSET_ENV: local
+      # If using production, comment development volume below
+      SUPERSET_ENV: development
     ports:
       - 8088:8088
-    command: "tail -f /dev/null"
     depends_on:
       - postgres
       - redis
     volumes:
-      - .:/home/work/incubator-superset
-      - superset-node-modules:/home/work/incubator-superset/superset/assets/node_modules
+      # this is needed to communicate with the postgres and redis services
+      - ./superset_config.py:/home/work/incubator-superset/superset/superset_config.py
+      # this is needed for development, remove with SUPERSET_ENV=production
+      - ../../superset:/home/work/incubator-superset/superset
 volumes:
   postgres:
     external: false
   redis:
     external: false
-  superset-node-modules:
+  frontend-dist:
```

Comment:
So after you rename to `frontend-dist`, where's `frontend-dist` volume? it will be empty?

Reply:
I forgot to remove it, it is not needed anymore